linux/secureboot_enable_disable/secureboot_enable_disable.yml

```diff
@@ -19,6 +19,15 @@
       block:
         - include_tasks: ../setup/test_setup.yml
 
+        - name: "Skip testcase '{{ ansible_play_name }}' for {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_major_ver }}"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip testcase '{{ ansible_play_name }}', it is not supported by {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_major_ver }}"
+            skip_reason: "Not Supported"
+          when:
+            - guest_os_ansible_distribution == 'MIRACLE'
+            - guest_os_ansible_distribution_major_ver | int == 8
+
```

Comment:
You can directly update `check_secureboot_support_status.yml`, which already defined not supported GOS list there